File: src/main/java/net/fenrir/blockphysics/BlockPhysics.java
package net.fenrir.blockphysics;

import net.fabricmc.api.ModInitializer;

import java.util.logging.Level;
import java.util.logging.Logger;

public class BlockPhysics implements ModInitializer {
    public static final String MOD_ID = "blockphysics";
    public static final String MOD_NAME = "Block Physics";


    @Override
    public void onInitialize() {
        Logger.getLogger(MOD_ID).log(Level.INFO, "[" + MOD_NAME + "]" + "Initialized");
    }
}

File: src/main/kotlin/net/fenrir/blockphysics/Blockphysics.kt
package net.fenrir.blockphysics

import net.fabricmc.api.ModInitializer
import org.slf4j.LoggerFactory

object Blockphysics : ModInitializer {
    private val logger = LoggerFactory.getLogger("blockphysics")

	override fun onInitialize() {
		// This code runs as soon as Minecraft is in a mod-load-ready state.
		// However, some things (like resources) may still be uninitialized.
		// Proceed with mild caution.
		logger.info("Hello Fabric world!")
	}
}